cluster/providers/utils/deploy_csi_driver_nfs.sh

```diff
@@ -26,3 +26,4 @@ volumeBindingMode: Immediate
 mountOptions:
 EOF
 
+kubectl create secret generic mount-options --from-literal mountOptions="nfsvers=4.1"
```

Comment:
how about `apply` with a yaml to resolve the issue seen in the CI?

Reply:
done